index.js

```diff
@@ -23,6 +23,7 @@ function OptimizeCssAssetsPlugin(options) {
   this.lastCallInstance = new LastCallWebpackPlugin({
     assetProcessors: [
       {
+        phase: 'compilation.optimize-chunk-assets',
```

Comment:
Please use `LastCallWebpackPlugin.PHASE.OPTIMIZE_CHUNK_ASSETS`.

Reply:
this didn't work ! the constant PHASE.OPTIMIZE_CHUNK_ASSETS is in the prototype (object) not in the exports

Reply:
unless we use `LastCallWebpackPlugin.prototype.PHASE.OPTIMIZE_CHUNK_ASSETS`

Reply:
My bad, i put it in the prototype by mistake.
Use `LastCallWebpackPlugin.prototype.PHASE.OPTIMIZE_CHUNK_ASSETS`, i will fix it later (with a change in the `LastCallWebpackPlugin`).